src/consumers/plain_event_consumer.cpp

```diff
@@ -0,0 +1,70 @@
+#include "consumers/plain_event_consumer.hpp"
+
+#include <iomanip>
+#include <iostream>
+
+using namespace events;
+
+uint64_t nanoseconds(time_point timestamp) {
+  return std::chrono::duration_cast<std::chrono::nanoseconds>(
+             timestamp.time_since_epoch())
+      .count();
+}
+
+void plain_event_consumer::consume(event const& e) {
+  std::visit(visitor, e);
+}
+
+std::string unescape(std::string const& s)
+{
+  std::string result;
+  for (char c : s)
```

Comment:
Ommitment of braces (in case of a multiline if else statement) does not seem to be optimal.